letsencrypt-auto

```diff
@@ -99,5 +110,9 @@ fi
 
 # Explain what's about to happen, for the benefit of those getting sudo
 # password prompts...
-echo "Running with virtualenv:" $SUDO $VENV_BIN/letsencrypt "$@"
-$SUDO $VENV_BIN/letsencrypt "$@"
+if [ -z "$SCL" ] ; then
+    echo "Running with virtualenv:" $SUDO $VENV_BIN/letsencrypt "$@"
+    $SUDO $VENV_BIN/letsencrypt "$@"
+else
+    echo "Now run: $SUDO $SCL \"$VENV_BIN/letsencrypt $@\""
```

Comment:
Maybe if people use `"$@"` rather than `$@` it should be `$SUDO $SCL \\"$VENV_BIN/letsencrypt "$@"\\"`, though I think that just defers expansion of shell arguments until further down the stack?